frontend/src/utilities/useAcceleratorProfileState.ts

```diff
@@ -65,7 +65,7 @@ const useAcceleratorProfileState = (
       } else {
         // check if there is accelerator usage in the container
         // this is to handle the case where the accelerator is disabled, deleted, or empty
-        const containerResourceAttributes = Object.values(ContainerResourceAttributes) as string[];
+        const containerResourceAttributes: string[] = Object.values(ContainerResourceAttributes);
         const possibleAcceleratorRequests = Object.entries(resources.requests ?? {})
           .filter(([key]) => !containerResourceAttributes.includes(key))
```

Comment:
Can we use `isEnumMember` here?

Reply:
Used `const possibleAcceleratorRequests = Object.entries(resources.requests ?? {})
          .filter(([key]) => isEnumMember(key, containerResourceAttributes))
          .map(([key, value]) => ({ identifier: key, count: value }));`